components/vf-progress-indicator/vf-progress-indicator.njk

```diff
@@ -0,0 +1,8 @@
+{# Make sure any variables are listed inside the following if statement #}
+<div class="vf-progress-indicator" style="width: {% if width %} {{width}} {% else %} 100% {% endif %}">
```

Comment:
Instead of hardcoding `width` here, the "vf ettiquete" would be to use a CSS custom prop here and in the CSS, like ` --vf-progress-indicator__width: 100%`

Happy to help with that if the method isn't clear?

Reply:
no need, implemented it